packages/pwa-kit-dev/src/configs/webpack/plugins.js

```diff
@@ -17,7 +17,7 @@ const pkg = require(resolve(projectDir, 'package.json'))
 const OVERRIDES_EXTENSIONS = '.+(js|jsx|ts|tsx)'
 
 const getOverridePath = (relativePath) => {
-    const extendPath = pkg?.ccExtensibility?.extends ? `node_modules/${EXT_EXTENDS}` : ''
+    const extendPath = EXT_EXTENDS ? `node_modules/${EXT_EXTENDS}` : ''
```

Comment:
`EXT_EXTENDS` actually no longer exists/defined. So is `EXT_OVERRIDES_DIR`.

I think we can trim down the implementation of `getOverridePath` and remove some lines of code.